notebooks/notebooks/iclr2022/estimate-flops-adv.ipynb

```diff
@@ -23,8 +23,18 @@
    "outputs": [],
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #51.                plt.axvline(x=xc, ls=":", lw=1, color=colors[3])
</pre><p>We do the same thing in <code>win-rate-vs-training.steps.ipynb</code> but it'd be more readable to use <code>linestyle</code> and <code>linewidth</code> instead of <code>ls</code> and <code>lw</code>    </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/HumanCompatibleAI/KataGoVisualizer/pull/53/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>